extensions/interactions/TextInput/directives/text-input-validation.service.ts

```diff
@@ -67,24 +67,27 @@ export class TextInputValidationService {
       });
     }
 
-    let isInt = function(n) {
-      return angular.isNumber(n) && n % 1 === 0;
+    let isInt = (n: number) => {
+      return ((typeof (n) === 'number') && (n % 1 === 0));
     };
 
     let rows = customizationArgs.rows.value;
     if (isInt(rows)) {
       let textSpecs = InteractionSpecsConstants.INTERACTION_SPECS.TextInput;
       let customizationArgSpecs = textSpecs.customization_arg_specs;
       let rowsSpecs = customizationArgSpecs[1];
-      let minRows = rowsSpecs.schema.validators[0].min_value;
-      let maxRows = rowsSpecs.schema.validators[1].max_value;
-      if (rows < minRows || rows > maxRows) {
-        warningsList.push({
-          type: AppConstants.WARNING_TYPES.ERROR,
-          message: (
-            'Number of rows must be between ' + minRows + ' and ' +
-            maxRows + '.')
-        });
+      let validators = rowsSpecs.schema.validators;
+      if (validators) {
```

Comment:
```suggestion
      if (validators.length == 2) {
```

Reply:
Done

Reply:
I tried this, it won't work because `min_value` and `max_value` are optional properties. We're only checking for `validators` to be present which itself is optional as well

Reply:
`if(validators && validators.length === 2)` seems more appropriate wdyt ?

Reply:
Yup, that lg!